docs/API.md

```diff
@@ -99,6 +100,28 @@ Make sure that the certificate you build with matches your `fcmSandbox` value.
 
 > Note: The integration between GCM and APNS is a bit finicky. Personally, I feel it is much better to send pushes to Android using GCM and pushes to iOS using APNS which this plugin does support.
 
+#### iOS VoIP Notifications
+
+It is possible to receive VoIP Notifications in iOS that can execute the "notification" event also when the application is in background or closed.
```

Comment:
I don't want to give people the false impression that the `on('notification')` event listener will fire when the app is closed. Yes, we can receive the notification but because the web view is not loaded in memory the JS won't run. Did you test this when the app is closed?

Reply:
Yes, I have tested it and it works. The JS is called even if the application is closed.

Reply:
Can that specific behaviour be backported to the normal notification case?

Reply:
No, it can. The only way to receive notifications with the app closed in iOS is through the VoIP notifications.